client/state/plugins/installed/actions.js

```diff
@@ -438,6 +443,10 @@ export function installPluginOnMultisite( siteId, plugin ) {
 	return installPluginHelper( siteId, plugin, true );
 }
 
+export function installPluginSimple( siteId, plugin ) {
+	return installPluginHelper( siteId, plugin, false, true );
+}
+
```

Comment:
It seems that `installPluginSimple` is avoiding plugin activation which is done through `activatePlugin`, can we instead change `installPlugin( siteId, plugin )` to accept a `activatePlugin` parameter? `simpleInstallation` is not descriptive enough and also devs may confuse it with Simple vs Atomic.